platform/src/generics/multiprecision_types/CMakeLists.txt

```diff
@@ -26,16 +26,16 @@ target_sources(RoughPy_Platform PRIVATE
         integer_conversion.h
         integer_number.cpp
         integer_number.h
-        float_type.cpp
-        float_type.h
-        float_arithmetic.cpp
-        float_arithmetic.h
-        float_comparison.cpp
-        float_comparison.h
-        float_conversion.cpp
-        float_conversion.h
-        float_number.cpp
-        float_number.h
+#        float_type.cpp
```

Comment:
Sanity checking: should all of these be commented out? if so should the files be removed too?